nullaway/src/main/java/com/uber/nullaway/autofix/Writer.java

```diff
@@ -0,0 +1,140 @@
+package com.uber.nullaway.autofix;
+
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.CompilationUnitTree;
+import com.sun.source.tree.Tree;
+import com.sun.tools.javac.code.Symbol;
+import com.uber.nullaway.Config;
+import com.uber.nullaway.ErrorMessage;
+import com.uber.nullaway.Nullness;
+import com.uber.nullaway.autofix.out.ErrorInfo;
+import com.uber.nullaway.autofix.out.Fix;
+import com.uber.nullaway.autofix.out.MethodInfo;
+import com.uber.nullaway.autofix.out.SeperatedValueDisplay;
+import com.uber.nullaway.autofix.out.TrackerNode;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import javax.lang.model.element.Element;
+
+public class Writer {
+  public static final String ERROR = "/tmp/NullAwayFix/errors.csv";
```

Comment:
Again, these static paths are probably going to be one of the main concerns about getting this into mainline NullAway...